<launch>
	<arg name="parts_list_filename" default="$(find o2as_parts_description)/config/assembly_parts_list.yaml"/>
	<include file="$(find o2as_vision)/launch/vision.launch">
	    <arg name="parts_list_filename" value="$(arg parts_list_filename)"/>
    </include>

    <node name="o2as_vision_demo" pkg="o2as_vision" type="o2as_vision_demo.py" output="screen">
    </node>
</launch>
